test/e2e/e2e.go

```diff
@@ -220,6 +223,10 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 	RunCleanupActions()
 }, func() {
 	// Run only Ginkgo on node 1
+
+	// Run test suite cleanup actions (if any) added. Called only once for test suite during teardown.
+	framework.RunSuiteCleanupActions()
```

Comment:
yeah we can use this mechanism to do cleanup like cluster de-registration from federation.